frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -413,6 +418,11 @@ export const surveyLogic = kea<surveyLogicType>([
             actions.reportSurveyEdited(survey)
             actions.loadSurveys()
         },
+        duplicateSurveySuccess: ({ survey }) => {
+            lemonToast.success(<>Survey {survey.name} is a duplicate copy, and can be edited now</>)
+            actions.loadSurveys()
+            router.actions.replace(urls.survey(survey.id))
```

Comment:
let's not re-route automatically, use a similar flow as here: https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/cohorts/cohortEditLogic.ts#L299 (i.e. a button to view new survey)